### ISSUE TITLE
add multi-validation mode?

### ISSUE THREAD

**Issue by bbernhard** [bbernhard/imagemonkey-core#34]
pretty interesting idea from a reddit user [1] 

"validating labels would be more efficient when displaying more images of, eg, cars at once and then simply select the positive examples. Like in those traffic sign captchas, but with multiple images."

Not sure if we already have tracked that in our issuses tracker, but might be something we could really consider? 

[1] https://www.reddit.com/r/MachineLearning/comments/75gqit/p_imagemonkey_whats_next/

**Comment by dobkeratops** [comment]
agree.  that might be a great way to deal with unlabelled images to - start by asking a very broad question ('highlight all the indoor images'..)

**Comment by bbernhard** [comment]
Just added a simple multi-validation mode. It's still in beta mode, as some things, mostly UI related, need to be improved, but one can already try it [here](https://imagemonkey.io/verify). The mode is per default disabled, but can be enabled by pressing 'm'. By pressing 'm' again it's possible to switch back to single validation mode again. In multi-validation mode it's also possible to quickly select multiple images by pressing down the left mousebutton and then swiping over the images. Here is a demo GIF which illustrates that: 

![multi-validation](https://user-images.githubusercontent.com/5710631/31565569-8b41989e-b067-11e7-9156-ef2c5233f76a.gif)

Just a quick note: The validation is done "strictly", which means that if the user forgets to select one picture it's as he would have pressed "No" in the single annotation mode. That's also the reason why I am limiting the number of pictures in the multi-validation mode to 8. I am afraid that If too many images (e.q 20) are loaded that some users won't have the patience to go through all of them and we will get a lot of false positives due to that. 

It's currently only possible to validate already labeled images, but can be extended to also validate unlabeled images in the future (as @dobkeratops proposed)

Any suggestions are welcome :)
